src/wav.c

```diff
@@ -1408,6 +1408,7 @@ wav_read_smpl_chunk (SF_PRIVATE *psf, uint32_t chunklen)
 		} ;
 
 	psf->instrument->basenote = note ;
+	psf->instrument->detune = (int8_t)(pitch / (0x40000000 / 25.0) + 0.5) ;
```

Comment:
Because I don't think I've used lrint before so I've looked up the manpage. It says that it rounds it depended on the current rounding direction. Is this state dependency something desirable?

Reply:
The rounding direction is set up in the FPU or MMX unit by the OS. It's always worked as I expected.